apps/web/cypress/e2e/multi-dwelling/multi-dwelling-combined-results.cy.ts

```diff
@@ -26,6 +27,8 @@ describe("multi dwelling: 9.9.9 and 9.10.14 walkthrough results", () => {
   });
 
   it("should display correct ui elements", () => {
+    cy.getByTestID(TESTID_BREADCRUMBS).should("be.visible");
```

Comment:
Adding check for the breadcrumbs as a whole exist. we test each individual section once in the multi-dwelling-walkthrough